Update education section 23/24

hamzaibrahim21 opened the pull request:
### Description
Update education section with a course and a seminar for winter semester 2023/2024
### TODO

- [x] Add course and seminar links
- [x] Add stefanie's email for registration
- [x] Add small paragraph for case-study seminar 
- [x] Add tag to previous courses
- [ ] Add github links to resources section (later)
### Status
- [x] Ready to go

AndreaVolkamer reviewed (changes requested):
@hamzaibrahim21 thanks for adding the content:
* for the new seminar, can you please use the content from LSF ('Bemerkungen' section).
* the seminar is new (not the same as the CADD seminar we offer in SS, can you add a new section for it please
* can you please order the course offers so that the newest one are listed on top, at present the WiSe23/24 ones, and add some tag 'previous' to the old ones.
* we can add the GH links later as you suggested

hamzaibrahim21 commented:
> @hamzaibrahim21 thanks for adding the content:
> 
> * for the new seminar, can you please use the content from LSF ('Bemerkungen' section).
> * the seminar is new (not the same as the CADD seminar we offer in SS, can you add a new section for it please
> * can you please order the course offers so that the newest one are listed on top, at present the WiSe23/24 ones, and add some tag 'previous' to the old ones.
> * we can add the GH links later as you suggested

@AndreaVolkamer Thanks for the feedback.
I added the paragraph , reordered offered courses and added previous tag.
If there is anything still missing, please let me know.
